src/components/grid-aware/VideoSpotlightBlock/VideoSpotlightBlock.stories.js

```diff
@@ -21,6 +21,7 @@ const Template = ({
     button={button}
     image={image}
     playButtonLink={playButtonLink}
+    blackBackground
```

Comment:
I would say that this should be a toggleable option in Storybook, so that you can easily view both variants.

Reply:
ok